build_tools/waf_dynamo.py

```diff
@@ -342,6 +331,9 @@ def default_flags(self):
         else:
             self.env.append_value(f, ['-DDM_PLATFORM_32BIT'])
 
+    if not hasattr(self, 'sdkinfo'):
```

Comment:
An optimization to not look for local tools too many times (took an extra 30s in total!)